RestaurantsNearMe/EnvironmentValues_Extensions.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  EnvironmentValues_Extensions.swift
+//  RestaurantsNearMe
+//
+//  Created by Colin Evans on 2023-11-25.
+//
+
+import Foundation
+import SwiftUI
+
+extension EnvironmentValues {
```

Comment:
These stubs were here until discussion around GeometryReader gets resolved. From what it seems, it doesn't seem like this way of designing on explicit landscape vs portrait really makes sense